apps/api/src/database/migrations/20230509134331_add_document_latest_version_column/migration.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[Document] ADD [latestVersionId] INT CONSTRAINT [Document_latestVersionId_df] DEFAULT 1;
```

Comment:
This shouldn't default to 1 - let's figure out if we can make it non-nullable and assign it within a transaction when creating the Document and DocumentVersion initially.

Reply:
@chrisdalykinandcarta should it not default to1?  As when a new doc is created it is always version 1